If we have readline, we don't use editline, so don't install it.

--- origsrc/heimdal-1.2.1/configure.in	2009-07-07 10:41:05.978840900 -0500
+++ src/heimdal-1.2.1/configure.in	2009-07-07 14:37:46.395243800 -0500
@@ -167,6 +167,7 @@ rk_CHECK_MAN
 rk_TEST_PACKAGE(readline,
 [#include <stdio.h>
  #include <readline.h>],-lreadline,,, READLINE)
+AM_CONDITIONAL(HAVE_READLINE, test "$with_readline" = yes)
 
 rk_TEST_PACKAGE(hesiod,[#include <hesiod.h>],-lhesiod,,, HESIOD)
 
--- origsrc/heimdal-1.2.1/lib/Makefile.am	2009-07-07 10:41:05.980090900 -0500
+++ src/heimdal-1.2.1/lib/Makefile.am	2009-07-07 14:38:18.124478700 -0500
@@ -17,7 +17,10 @@ endif
 if !HAVE_OPENSSL
 dir_hcrypto = hcrypto
 endif
+if !HAVE_READLINE
+dir_editline = editline
+endif
 
-SUBDIRS = roken vers editline $(dir_com_err) sl wind asn1 \
+SUBDIRS = roken vers $(dir_editline) $(dir_com_err) sl wind asn1 \
 	$(dir_hcrypto) hx509 krb5 ntlm kafs gssapi hdb kadm5 \
 	auth $(dir_45) $(dir_otp) $(dir_dce)
